youths/models.py

```diff
@@ -73,3 +82,7 @@ def __str__(self):
             self.profile.user.last_name,
             self.profile.user.uuid,
         )
+
+    def save(self, *args, **kwargs):
+        self.full_clean()
+        return super(YouthProfile, self).save(*args, **kwargs)
```

Comment:
in python 3 you can just do `super().save(*args, **kwargs)` 🙂

Reply:
That's very mucho true 🤝